src/components/Navbar.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import logo from "../utilities/logo.png";
+import { NavLink } from "react-router-dom";
+
+function Navbar() {
+  const [link, setLink] = useState(null);
+
+  const handleClick = (id) => {
+    setLink(id);
+  };
+
+  return (
+    <div className="navbar navbar-light bg-light">
+      <div className="container">
+        <div className="d-flex gap-4 align-items-center">
+          <img style={{ width: "50px" }} src={logo} alt="" />
+          <h4>Space Traveler's Hub</h4>
+        </div>
+        <div className="d-flex gap-4 links-container">
+          <NavLink  className={ link === 1? "underline" : "" } onClick={() => handleClick(1)} to="/rockets">
+            Rockets
+          </NavLink>
+          <NavLink className={ link === 2? "underline" : "" } onClick={() => handleClick(2)} to="/missions">
+            Missions
+          </NavLink>
+          <span>| </span>
+          <NavLink className={ link === 3? "underline" : "" } onClick={() => handleClick(3)} to="myprofile">
+            My profile
```

Comment:
Great implementation of the Navlink and the handle click function.
however, it is not necessary to implement the handle click function because Navlink already has a functionality and add the .active class on every link